plugins/inputs/docker_log/docker_log.go

```diff
@@ -329,21 +365,25 @@ func tailStream(
 	r := bufio.NewReaderSize(reader, 64*1024)
 
 	var err error
-	var message string
+	var message []byte
+	var messageTs time.Time
+
 	for {
-		message, err = r.ReadString('\n')
+		message, err = r.ReadBytes('\n')
+		//Parsing timestamp and message
+		messageTs, message = parseLogEntry(message, &acc, &containerID)
```

Comment:
Change this function to support returning an error, if an error is returned you can `acc.AddError(err)` and skip processing the line.